.github/workflows/buildwin.yml

```diff
@@ -1,17 +1,14 @@
 name: Build z390 (Windows)
-on:
-  push:
-    branches:
-      - 'main'
-      - 'feature/**'
-      - '!**-nobuild'
+on: [push]
 
 jobs:
   build:
     runs-on: windows-2019
     steps:
       - name: Checkout main
         uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: Build z390 (win)
         shell: cmd
         run: call .\BUILD.BAT
```

Comment:
I assume this is what ran in the problem I saw yesterday when I did the "Fetch upstream". Not sure why the windows version was run. Why not the Linux/Unix version? (Rhetorical question only.)

Reply:
So this branch has not been merged yet so that's why you did not see the Linux build. The windows one currently runs on push to main which is what happened in your repo when you fetched the upstream.